gio-subclass/src/auto/action.rs

```diff
@@ -123,7 +125,7 @@ unsafe extern "C" fn action_get_name<T: ObjectType>
             gobject_ffi::g_object_set_qdata_full(gptr as *mut gobject_ffi::GObject,
                 glib_ffi::g_quark_from_string("rs_get_name".to_glib_none().0),
                 ret as *mut c_void,
-                None //TODO: how do we free the data
+                Some(<String as FreeGlibPtr>::free)
```

Comment:
I would simply define a local function here that does the freeing instead of defining a new trait

Reply:
Oh I see the problem... hm!